src/ray/raylet/node_manager.cc

```diff
@@ -1191,11 +1201,6 @@ void NodeManager::HandleNotifyGCSRestart(rpc::NotifyGCSRestartRequest request,
 
 bool NodeManager::UpdateResourceUsage(const NodeID &node_id,
                                       const rpc::ResourcesData &resource_data) {
-  // Trigger local GC at the next heartbeat interval.
```

Comment:
Not sure why it's added here. It seems being added by gcs_actor_scheduler which is not enabled by default. I don't think it's the right place to add this. Delete and check CI tests.